go.mod

```diff
@@ -15,19 +15,14 @@ require (
 )
 
 replace (
-	// Use v0.8.10 of hcsshim to fix nancy alert.
 	github.com/Microsoft/hcsshim v0.8.7 => github.com/Microsoft/hcsshim v0.8.10
-	// Apply fix for CVE-2020-15114 not yet released in github.com/spf13/viper.
```

Comment:
I also removed the comments when the replace is due to nancy. 

As this is the usual reason. I've kept them for other reasons like our CAPI forks.